site/_posts/2018-07-23-calling-an-embedded-ansible-playbook-from-the-vm-provision-state-machine.md

```diff
@@ -1,14 +1,14 @@
----     
-title: Calling an Embedded Ansible Playbook from the VM Provision State Machine 
+---
+title: Calling an Embedded Ansible Playbook from the VM Provision State Machine
 date: 2018-07-23
-tags: ansible cloudforms automate
+tags: ansible cloudforms automate embedded-ansible
```

Comment:
@Fryguy so what does the `ansible` tag represent, tower?

Reply:
Just ansible in general (could be tower, embedded ansible, built-in roles, etc.)